### PR TITLE
Update flavors.md

### PR DESCRIPTION
**Pull request by sarbogast**
I added a link to a tutorial that I wrote describing the full process of flavorizing a Flutter project including support for web, Firebase and FlutterFire CLI.

**IMPORTANT:** Due to work on the docs.flutter.dev infrastructure, this repo is **not accepting pull requests**.

Instead of creating a PR, please file an issue (https://github.com/flutter/website/issues/new/choose) about the needed change.

We expect to start accepting PRs again the week of May 11.

### CODE REVIEW

**Comment by khanhnwin** [comment]
Hey @sarbogast - this repo is currently frozen and not accepting any PRs. We'll start accepting PRs again the week of May 11. Apologies for the inconvenience. Thanks!

**Comment by khanhnwin** [comment]
Hey @sarbogast can you please bring this up to date with main? 

@sfshaza2 can you take a look for content review?

**Comment by domesticmouse** [comment]
PTAL @sfshaza2

**Comment by sarbogast** [comment]
@khanhnwin I'm sorry but I'm not sure what you mean by that or how to do it.

**Review by sfshaza2 — APPROVED** [review]
lgtm

File: src/deployment/flavors.md [hunk 1]
```diff
@@ -29,5 +32,6 @@ The following packages are listed alphabetically:
 [Flutter Ready to Go]: {{site.medium}}/flutter-community/flutter-ready-to-go-e59873f9d7de
 [Build flavors in Flutter (Android and iOS) with different Firebase projects per flavor]: {{site.medium}}/@animeshjain/build-flavors-in-flutter-android-and-ios-with-different-firebase-projects-per-flavor-27c5c5dac10b
 [Flutter 1.17 — no more Flavors, no more iOS Schemas. Command argument that changes everything]: {{site.medium}}/@tatsu.ukraine/flutter-1-17-no-more-flavors-no-more-ios-schemas-command-argument-that-solves-everything-8b145ed4285d
+[Multi-environment Flutter Projects with Flavors]: {{https://sebastien-arbogast.com/2022/05/02/multi-environment-flutter-projects-with-flavors/
```
Comment (domesticmouse) [inline src/deployment/flavors.md:35]: I believe the `{{` at the start of the URL is incorrect. Please remove. 

For context, we use `{{variable.name}}` to interpolate site wide variables in the publishing engine.

**Review by domesticmouse — CHANGES_REQUESTED** [review]

File: src/deployment/flavors.md (hunk 1 above)
Comment (sarbogast) [reply src/deployment/flavors.md]: Thanks. I fixed it.

File: src/deployment/flavors.md [hunk 2]
```diff
@@ -13,7 +13,10 @@ These articles address flavors for both iOS and Android.
 * [Flavoring Flutter][]
 * [Flutter Ready to Go][]
 * [Build flavors in Flutter (Android and iOS) with different Firebase projects per flavor][]
-* [Flutter 1.17 — no more Flavors, no more iOS Schemas. Command argument that changes everything][]
+* [Flutter 1.17 — no more Flavors, no more iOS Schemas.
+   Command argument that changes everything][]
```
Comment (sfshaza2) [inline src/deployment/flavors.md:17]:
```suggestion
  Command argument that changes everything][]
```

**Review by sfshaza2 — APPROVED** [review]
lgtm!

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Review by domesticmouse — APPROVED** [review]

**Review by khanhnwin — APPROVED** [review]

**Pull request merged**